astroplan/constraints.py

```diff
@@ -394,7 +395,79 @@ def compute_constraint(self, times, observer, targets):
                              "MoonSeparationConstraint.")
         return mask
 
+    
+class DarkGreyBrightConstraint(Constraint):
```

Comment:
What's the best name for this constraint. Opinions @ejeschke, @eteq, @adrn?

Some alternative suggestions: 
* `MoonBrightnessConstraint` (which is ambiguous given that we already have a `MoonIlluminationConstraint)
* `MoonContaminationConstraint` (which is too long to type)
* `DarkTimeConstraint` (which is shorter to type but more vague)